src/classes/RD2_DataMigration_TEST.cls

```diff
@@ -1934,6 +2008,29 @@ private class RD2_DataMigration_TEST {
         );
     }
 
+    /**
+     * @description Sets store errors in error settings to false
+     * @return void
+     */
+    private static void setStoreErrorsInErrorSettingsToFalse() {
+        setStoreErrorsInErrorSettings(false);
+
+    }
+
+    /**
+     * @description Sets store settings to enable or disable in error settings
+     * @param isEnabled true or false
+     * @return void
+     */
+    private static void setStoreErrorsInErrorSettings(Boolean isEnabled) {
+        UTIL_CustomSettingsFacade.getErrorSettingsForTests(
+            new Error_Settings__c(
+            Error_Notifications_On__c = isEnabled,
```

Comment:
`Error_Notifications_On__c` regulates email/chatter notification while we need to verify `Store_Errors_On__c`. Thus, let's keep error notification field the same (`false` ?) regardless of the store error field config.